requirements_dev_optional.txt

```diff
@@ -16,6 +17,6 @@ flake8==3.9.2
 pytest-cov==2.12.0
 pytest-doctestplus==0.4.0
 h5py==2.10.0
-s3fs==2021.5.0; python_version > '3.6'
-fsspec==2021.5.0; python_version > '3.6'
-moto[server]>=1.3.14; python_version > '3.6'
+s3fs==2021.5.0; python_version > '3.7'
+fsspec==2021.5.0; python_version > '3.7'
+moto[server]>=1.3.14; python_version > '3.7'
```

Comment:
I think we can just drop `python_version`. Believe this was just here because these packages didn't exist for Python 3.6